griptape/drivers/embedding/sagemaker_embedding_driver.py

```diff
@@ -0,0 +1,35 @@
+import json
+import boto3
+from typing import Any
+from attr import define, field, Factory
+from griptape.drivers import BaseEmbeddingDriver
+
+
+@define
+class AmazonSagemakerEmbeddingDriver(BaseEmbeddingDriver):
+    endpoint: str = field(kw_only=True)
+    session: boto3.Session = field(
+        default=Factory(lambda: boto3.Session()), kw_only=True
+    )
+    sagemaker_client: Any = field(
+        default=Factory(
+            lambda self: self.session.client("runtime.sagemaker"),
```

Comment:
Why `runtime.sagemaker` over `sagemaker-runtime`? Both appear to work, but we use `{service}-runtime` in other places of the framework.